src/calculateRentalCost.js

```diff
@@ -4,7 +4,19 @@
  * @return {number}
  */
 function calculateRentalCost(days) {
-  // write code here
+  if (days >= 3 && days < 6) {
+    return 40 * days - 20;
```

Comment:
You've hard-coded the values `3`, `6`, `20`, and `40`. It's better to extract these into well-named constants to increase readability and maintainability of your code. For instance, you could create constants like `SHORT_TERM`, `MEDIUM_TERM`, `LONG_TERM`, `SHORT_TERM_DISCOUNT`, `MEDIUM_TERM_DISCOUNT`, `LONG_TERM_DISCOUNT`, and `DAILY_RATE`.